parquet-thrift/src/main/java/org/apache/parquet/thrift/projection/deprecated/PathGlobPattern.java

```diff
@@ -20,8 +20,8 @@
 
 import org.apache.hadoop.fs.GlobPattern;
 
-import java.util.regex.Pattern;
-import java.util.regex.PatternSyntaxException;
+import com.google.re2j.Pattern;
```

Comment:
I think this may not work for projects like Spark who are using Hadoop shaded client, since the `GlobPattern.compiled` is relocated to `org.apache.hadoop.shaded.com.google.re2j.Pattern`.

It might be easier to just remove the class as it has been marked as deprecated since Parquet 1.8.0, 2015

Reply:
+1 for cutting. i will update the patch